meetings/2023-10-18.md

```diff
@@ -42,15 +54,42 @@
+* eP: it would be useful to have signals of intent to implement (simple web form, as discussed yesterday on the chat).
+* SC: There is an index of applications and servers: https://solidproject.org/apps , https://solidproject.org/developers/tutorials/getting-started#own-server . The QA focuses on differentiating *conforming* from *work in progress*. It does not signal anything beyond that. If we were to invest more time for this kind of things, I want to make sure that there is synergy with what the Solid Team is doing, not repeating work.
+* eP: It is nice to have a basic list, but it does not tell much. Do the listed server implement storage, idp, authorisation, notification? We need a finer product class granularity.
+* SC: And that part belongs to the QA, and is already covered. Each collected report lists all the test-cases that an implementation passes.
+* eP: AC, I recall that you have written a test-suite for the issuer (OP aka. IDP). Is there an equivalent test for the client?
+* AC: only for the OP. Would be more complicated to make one for clients, but that would be useful for developers.
+* SC: we need to take Solid QA seriously, it is at the core of what we are doing. We need to prove that implementations are conforming with the specification. This is not a requirement for CGs. This work was for the WG to be able to advance on this, and prevent objections. Please let's create more energy around that.
 
 
 ### Joint Solid CG and Social CG meeting
 URL: https://lists.w3.org/Archives/Public/public-solid/2023Oct/0036.html
 
 * SC: Thread also at https://lists.w3.org/Archives/Public/public-swicg/2023Oct/0060.html
 * SC: There is interest from participants in both groups. Let's make it so...
+* SC: for example, the Linked Data Notification developed by the Social Web WG was done with collaboration from MIT at the time, and is compatible (and used) by both Solid and the Social Web people.
+* SC: the work of each group is not conflicting with the other. E.g. Solid Protocol is about client-server interaction, while ActivityPub focuseS on server-server federation. A given server could possibly implement both.
 * SC: SocialCG's meetings are on Friday. Our Tuesday (Special Topic Meeting slot) may be best.
 * SC: Any thoughts, suggestions, concerns?
+* eP: it might be useful to have a short pad for drafting topics, so that both side can prepare the meeting. Would make the meeting more productive. Example: authentication/authorization, media-types. About the server-server interaction: one of the server sometimes just acts as a client. All this could be prepared in advance.
+* SC: we did this for WebAgents. I agree we can do that. Authentication is a tricky topic: there are at least 4 ways how it is done (Fediverse, IndieAuth, Solid, ActivityPub). 
```

Comment:
```suggestion
* eP: It might be useful to have a short pad for drafting topics, so that both sides can prepare for the meeting. Would make the meeting more productive. Example: authentication/authorization, media-types. About the server-server interaction: one of the server sometimes just acts as a client. All this could be prepared in advance.
* SC: We did this for WebAgents. I agree we can do that. Authentication is a tricky topic; there are at least 4 ways it is done (Fediverse, IndieAuth, Solid, ActivityPub). 
```